Add files via upload

watsonsd opened the pull request.

watsonsd committed:
Add files via upload

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Ed-Fi-Exchange-OSS/SDCOE-Universal-Transcript?pullRequest=1) <br/>All committers have signed the CLA.

watsonsd committed:
Initial application commit to Exchange JWatson

watsonsd committed:
Completed remaining TO DO items 9-8-2021

1. I've added a new more extensive CLI Reference document to the /docs folder (MSDF SDCOE Ed-Fi Transcript Prototype Backend and CLI Reference v1.0.md") and updated a similar named PDF document ("MSDF SDCOE Ed-Fi Transcript Prototype Backend and CLI Reference.pdf") with a note that the new markdown document has more information and should be located in the same folder.

2. I removed the unneeded /design folder from my fork of the project.

3. I also removed the "SDCOE-Universal-Transcript/web-client/src/assets/fonts/proxima_nova" folder.
